pkg/apis/routing/v1alpha1/interfaces.go

```diff
@@ -26,50 +26,50 @@ import (
 	"knative.dev/pkg/kmeta"
 )
 
-// Router is implemented by all Routing types.
-type Router interface {
+// TMComponent is implemented by all TriggerMesh components.
+type TMComponent interface {
```

Comment:
After longer reflection, I think I like `Reconcilable` a lot more than `TMComponent`, because it's exactly what this interface exists for.

Feel free to use this thread to voice your opinions.

Reply:
+1 on `Reconcilable`

Reply:
Renamed to `Reconcilable` 👍